src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -612,7 +612,7 @@ export function getAlterationEnrichmentColumns(
             ),
             sortBy: (d: AlterationEnrichmentRow) => Number(d.logRatio),
             download: (d: AlterationEnrichmentRow) =>
-                formatLogOddsRatio(d.logRatio!),
+                d.logRatio ? formatLogOddsRatio(d.logRatio) : '-',
```

Comment:
@dippindots is the "-" what we use elsewhere for empty?

Reply:
@alisman Not sure if we use `-` globally. In this case, we use `-` in render method for empty: https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/resultsView/enrichments/EnrichmentsUtil.tsx#L605